spec/features/carrier_spec.rb

```diff
@@ -1,11 +1,43 @@
 # frozen_string_literal: true
 
-RSpec.describe 'Carrier' do
+RSpec.describe 'Carrier USER role' do
+  let(:carrier) { carriers(:carrier) }
+  let(:user) { users(:user) }
+
+  before do
+    visit '/'
+    sign_in user
+  end
+
+  scenario 'SHOW' do
+    visit carriers_path
+    click_link carrier.name
+
+    expect(page).to have_content('test carrier')
+    expect(page).to have_content('babywearing')
+    expect(page).to have_content('test model')
+  end
+
+  scenario 'CREATE' do
```

Comment:
I believe this would be 'NEW', not 'CREATE' as the user has not yet submitted the form to perform the create action.